compiler/src/test/kotlin/utility.kt

```diff
@@ -0,0 +1,104 @@
+package dk.aau.cs.d409f19
+
+import dk.aau.cs.d409f19.cellumata.CompilerData
+import dk.aau.cs.d409f19.cellumata.ast.*
+import dk.aau.cs.d409f19.cellumata.visitors.LiteralExtractorVisitor
+import dk.aau.cs.d409f19.cellumata.visitors.ScopeCheckVisitor
+import dk.aau.cs.d409f19.cellumata.visitors.TypeChecker
+import org.antlr.v4.runtime.CharStreams
+import org.antlr.v4.runtime.CommonTokenStream
+
+class Utilities {
+
+    companion object {
+
+        /**
+         * Compiles a program given as string parameter
+         */
+        fun compileProgram(program: String): CompilerData {
```

Comment:
Why is all the utility functions in a companion object? Kotlin allows for lose functions that isn't part of a class

Reply:
I was taught by @NicEastvillage, that to reference functions statically, like `compileProgram(string)` they would have to be a companion object. Any other way?

Reply:
see `ast/reduce.kt`

Reply:
Yes, good call. Brainfart on my behalf. Fixed in 4b4447b, resolve if sufficient.